internal/command/command.go

```diff
@@ -104,30 +104,20 @@ func (c Command) Display(data TemplateData) (string, error) {
 	return strings.Join(str, ": "), nil
 }
 
-// execute runs the command with the given config and outputs.
-func (c Command) execute(ctx context.Context, config *Config, args Args, previousOutputs Outputs, streams *genericclioptions.IOStreams) (Outputs, error) {
-	outputs := Outputs{}
-
-	for k, v := range previousOutputs {
-		outputs[k] = v
-	}
-
+// Execute runs the command with the given config and outputs.
+func (c Command) Execute(ctx context.Context, config *Config, args Args, outputs Outputs, streams *genericclioptions.IOStreams) (Outputs, error) {
 	data := TemplateData{
 		LocalPort: config.LocalPort,
 		Args:      args,
-		Outputs:   previousOutputs.Stdout(),
+		Outputs:   outputs,
 	}
 
 	cmd, err := c.ToCmd(ctx, data)
 	if err != nil {
 		return nil, err
 	}
 
-	cmdStr, err := c.Display(data)
-	if err != nil {
-		return nil, err
-	}
-
+	cmdStr, _ := c.Display(data)
```

Comment:
why ignore these errors and the ones below?

Reply:
They are unreachable branches. The first call to Args is the only reachable one. This suggests some further refactoring.